build/cmake/DefineInstallationPaths.cmake

```diff
@@ -22,10 +22,11 @@
 set(BIN_INSTALL_DIR "bin" CACHE PATH "The binary install dir (default: bin)")
 set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}" CACHE PATH "The library install dir (default: lib${LIB_SUFFIX})")
 set(INCLUDE_INSTALL_DIR "include" CACHE PATH "The library install dir (default: include)")
-set(CMAKE_INSTALL_DIR "cmake" CACHE PATH "The subdirectory to install cmake config files (default: cmake)")
+set(CMAKE_INSTALL_DIR "lib/cmake" CACHE PATH "The subdirectory to install cmake config files (default: cmake)")
```

Comment:
Should this be share/cmake?

Reply:
@jeking3 As I know cmake module can be install on lib/cmake like facebook library [wangle](https://github.com/facebook/wangle) and [folly](https://github.com/facebook/folly) etc.

For detailed information you can see https://gitlab.kitware.com/cmake/community/wikis/doc/tutorials/How-to-create-a-ProjectConfig.cmake-file

Reply:
If LIB_SUFFIX is defined, should it be ${LIB_INSTALL_DIR}/cmake perhaps to be consistent?